_posts/2020-11-20-status-update.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "2020-11-20: IAC Team Status Update"
+author: david22swan
+categories:
+  - team
+  - status
+tags:
+---
+
+Coming at you with the latest weekly blog post, here to fill you in on all the juicy IAC gossip!
+
+## Gitpod
+
+As of a few days ago a pr ([pdk-templates#354][pdk-templates-354]), submitted by the execellent [logicminds][logicminds] has been merged in, adding support for gitpod puppet development to the pdk-templates as an opt in option, allowing contributors to edit any enabled module via VSCode from the browser without requiring any local setup or install.
```

Comment:
```suggestion
As of a few days ago a pr ([pdk-templates#354][pdk-templates-354]), submitted by the excellent [logicminds][logicminds] has been merged in, adding support for gitpod puppet development to the pdk-templates as an opt in option, allowing contributors to edit any enabled module via VSCode from the browser without requiring any local setup or install.
```

Reply:
Done

Reply:
Maybe say that,  a container is creating the gitpod environment that comes packed with pdk, puppet-debugger, puppet-agent, zsh, oh-my-zsh plugins and other productivity improvements to enhance puppet development.